### PR TITLE
Zib2020 BowelFunction

### PR DESCRIPTION
**Pull request by LuudSlagter**

### CODE REVIEW

File: ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-BowelFunction-02.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            <display value="Zachte stukjes met gehavende randen"/>
+         </coding>
+      </valueCodeableConcept>
+   </component>
+   <component>
+      <code>
+         <coding>
+            <system value="http://loinc.org"/>
+            <code value="9397-1"/>
+            <display value="Color of Stool"/>
```
Comment (Ashwin-nictiz) [inline ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-BowelFunction-02.xml:75]: There is a Dutch translation for this code.

"Kleur"

File: ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-BowelFunction-01.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            <display value="Als een worst of slang, glad en zacht"/>
+         </coding>
+      </valueCodeableConcept>
+   </component>
+   <component>
+      <code>
+         <coding>
+            <system value="http://loinc.org"/>
+            <code value="9397-1"/>
+            <display value="Color of Stool"/>
```
Comment (Ashwin-nictiz) [inline ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-BowelFunction-01.xml:63]: There is a Dutch translation for this code.

"Kleur"

File: ada_2_fhir-r4/zibs2020/payload/0.8-dev/nl-core-BowelFunction.xsl [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                    </component>
+                </xsl:for-each>
+                
+                <xsl:for-each select="defecatie_kleur">
+                    <component>
+                        <code>
+                            <coding>
+                                <system value="http://loinc.org"/>
+                                <code value="9397-1"/>
+                                <display value="Color of Stool"/>
```
Comment (Ashwin-nictiz) [inline ada_2_fhir-r4/zibs2020/payload/0.8-dev/nl-core-BowelFunction.xsl:136]: There is a Dutch translation for this code.

"Kleur"

File: ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-BowelFunction-02.xml (hunk 1 above)
Comment (LuudSlagter) [reply ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-BowelFunction-02.xml:75]: This is just a partial translation of the LOINC concept (a full translation would be something like 'Kleur van feces'), so I'd rather not use that translation.

**Pull request merged**